### ISSUE TITLE
codecTest

### ISSUE THREAD

**Issue by a897456** [facebookresearch/AudioDec#39]
@bigpon Hi bigpon
During inference, can other code rates be achieved by reducing the number of codebook_num?

**Comment by bigpon** [comment]
Yes, you can.
However, since we didn't applied the codebook dropout during training, the performance might be significantly dropped.
You can either implement the dropout logics in the training or directly retrain/fine-tune the models with the desired number of codebooks.
